src/main.js

```diff
@@ -4,14 +4,17 @@ import {render} from './framework/render.js';
 import {RenderPosition} from './framework/render.js';
 import BoardPresenter from './presenter/event-board-presenter.js';
 import EventsModel from './model/events-model.js';
+import {generateFilter} from './mock/filter.js';
 
 const siteHeader = document.querySelector('.trip-main');
 const siteFilter = siteHeader.querySelector('.trip-controls__filters');
 const siteEvents = document.querySelector('.trip-events');
 
 const eventsModel = new EventsModel();
 const boardPresenter = new BoardPresenter(siteEvents, eventsModel);
+const filters = generateFilter(eventsModel.events);
+window.console.log(filters);
```

Comment:
А почему не просто?)
```suggestion
console.log(filters);
```
Ну и желательно с кода на проверку это все вычищать